configuration/src/test/java/io/airlift/configuration/TestMonitor.java

```diff
@@ -18,17 +18,18 @@
 import com.google.common.base.Joiner;
 import com.google.inject.spi.Message;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Collection;
+import java.util.LinkedHashSet;
+import java.util.Set;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.fail;
 
 class TestMonitor
         implements Problems.Monitor
 {
-    private final List<Message> errors = new ArrayList<>();
```

Comment:
Why? Seems unrelated. Changes order of error messages.

Reply:
this class gets deleted in later commit, but when I've added changes to Problems in commit no. 2, then it sometimes contained duplicate messages